packages/manager/apps/key-management-service/src/hooks/guide/guidesLinks.constant.ts

```diff
@@ -2,6 +2,11 @@ import { CountryCode } from '@ovh-ux/manager-config';
 
 type GuideLinks = { [key in CountryCode]: string };
 
+export const SUPPORT_URL: Record<string, string> = {
```

Comment:
```suggestion
export const SUPPORT_URL: Record<CountryCode, string> = {
```

Reply:
Region like `EU` for example are not `CountryCode`